core/trino-server/src/main/provisio/trino.xml

```diff
@@ -73,13 +73,12 @@
         </artifact>
     </artifactSet>
 
-    <!--
```

Comment:
Wrong commit for these changes

I recommend reordering this commit so it's first, as that makes amending the other one easier